### PR TITLE
Add more Knative repos

### PR DESCRIPTION
**Pull request by aliok**
Add more Knative repos, per https://github.com/knative/test-infra/issues/3632#issuecomment-1411732127

### CODE REVIEW

**Comment by aliok** [comment]
@tegioz 

How can I add the subprojects in Knative that live outside of github.com/knative GitHub organization?

Some examples:
- https://github.com/knative-sandbox/eventing-kafka-broker/
- https://github.com/knative-sandbox/reconciler-test
- ...

Would this work:

```
    - name: sandbox/eventing-kafka-broker
      url: https://github.com/knative-sandbox/eventing-kafka-broker
      check_sets:
        - code-lite
```

Also, is there a way to exclude these from the health reports but have them in CLOTributor?

**Comment by tegioz** [comment]
> Would this work:
> 
> ```
>     - name: sandbox/eventing-kafka-broker
>       url: https://github.com/knative-sandbox/eventing-kafka-broker
>       check_sets:
>         - code-lite
> ```

Yes, you can add more repositories that way 👍

> Also, is there a way to exclude these from the health reports but have them in CLOTributor?

No, not at the moment. But this is an interesting request, we'll make this possible 🙂

**Comment by aliok** [comment]
> Yes, you can add more repositories that way 👍

Thanks, I will create a separate PR for that.


> No, not at the moment. But this is an interesting request, we'll make this possible 🙂

I can be the tester for that feature, if you would need a hand :)

My motivation for that feature request is:
- I don't want to create noise in the https://clomonitor.io/projects/cncf/knative page by adding dozens of repositories there
- Though I want their `good-first-issue`s to be listed in CLOTributor.

**Comment by tegioz** [comment]
> I can be the tester for that feature, if you would need a hand :)

Sure, that'd be great, thanks 🙂

> My motivation for that feature request is:
> 
> * I don't want to create noise in the https://clomonitor.io/projects/cncf/knative page by adding dozens of repositories there
> * Though I want their `good-first-issue`s to be listed in CLOTributor.

Yes, that makes sense 👍

**Comment by tegioz** [comment]
#1074 has been deployed @aliok 🙂

**Comment by aliok** [comment]
@tegioz you're so fast! Thanks for adding this feature so quickly.

Let me send a PR for the other Knative repos with `exclude: \n - clomonitor`

**Comment by tegioz** [comment]
Cool, no worries!

File: data/cncf.yaml [hunk 1]
```diff
@@ -1010,6 +1010,26 @@
       url: https://github.com/knative/serving
       check_sets:
         - code
+    - name: test-infra
+      url: https://github.com/knative/test-infra
+      check_sets:
+        - code-lite
```
Comment (dprotaso) [inline data/cncf.yaml:1016]: This should be `knative/infra` now

Comment (aliok) [reply data/cncf.yaml:1016]: created https://github.com/cncf/clomonitor/pull/1264 now

**Pull request merged**